sdk/servicebus/azure-servicebus/azure/servicebus/aio/management/_management_client_async.py

```diff
@@ -407,12 +410,12 @@ async def create_queue(  # pylint: disable=too-many-locals
         :rtype: ~azure.servicebus.management.QueueProperties
         """
         forward_to = _normalize_entity_path_to_full_path_if_needed(
-            forward_to, self.fully_qualified_namespace
+            self.fully_qualified_namespace, forward_to
         )
         forward_dead_lettered_messages_to = (
             _normalize_entity_path_to_full_path_if_needed(
-                forward_dead_lettered_messages_to,
                 self.fully_qualified_namespace,
+                forward_dead_lettered_messages_to
```

Comment:
just want to double-check that the re-ordering of arguments was intentional here

Reply:
yes! I changed the order of this to allow for =None on forward_to